### ISSUE TITLE
Arcface training strategy(Resnet50, CASIA - Webface, batch size < 128)

### ISSUE THREAD

**Issue by ghost** [ronghuaiyang/arcface-pytorch#100]
im trying to train insightFace on CASIA - Webface, and i tryed to train arcface and cosface from scratch. Due my limit system, i just set batch size to 64. My result when i training arcface, cosface on LFW are 99,3 and 99,4 respectively, not like result in the paper.

could anyone give me some advice on how to train arcface, learning rate...?

Configs for arc/cosface are same, like this:

#hyper params
weight_decay: 5e-4
logits_scale: 30
logits_margin: 0.5 ( 0.35 for cosface)
drop_ratio: 0.5
##-------------------
#optimizer
optimizer: SGD
momentum: 0.9
base_lr : 0.01
lr_steps: [20000, 28000]

Thanks in advance!

**Comment by jack-gits** [comment]
what's the value of train accuracy and train loss?

**Comment by jack-gits** [comment]
I'm training arcface, base lr is 0.1

**Comment by ghost** [comment]
> what's the value of train accuracy and train loss?
 
I did not monitor the accuracy of the training, my train losses end at 2.7 (arcface) and 1.7 (cosface).
have u got better results?

**Comment by jack-gits** [comment]
still in training. worse than you

**Comment by ghost** [comment]
> still in training. worse than you

refer https://github.com/auroua/InsightFace_TF,
i think u can improve acc by set smaller base lr while batch size is set small.
